.yamato/training-int-tests.yml

```diff
@@ -12,6 +12,7 @@ test_mac_training_int_{{ editor.version }}:
   variables:
     UNITY_VERSION: {{ editor.version }}
   commands:
+    - pip install torch --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
```

Comment:
BTW this _shouldn't_ be necessary for the code to run; normally if you don't have `torch` it'll just default to TF.

Reply:
I am facing some issues with the import torch statement. The yamato tests need to have torch (just like circleci does).

Reply:
Wrapping the import statement in try/except for torch saver should fix this. This is missing in saver and I wasn't aware of that.

Reply:
We import torch at a lot of places, we already add torch to the test_requirements.txt, I think it is fair to also import it for fast training. Ignore the changes to this file, this was a mistake. I needed to modify yamato_utils.py when creating the virtual environment instead

Reply:
IMO there's some benefit to not having it in for the Yamato tests. The code should run without `torch` at least for the next release.

Reply:
The cause of this has to be fixed anyway, otherwise can't run with environment with TF only. Commented below.